pages/glossary/browsing-context-related-nodes.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Set of browsing context related nodes
+key: browsing-context-related-nodes
+---
+
+Two HTML [nodes](https://dom.spec.whatwg.org/#concept-node) are _browsing context related_ if:
+
+- their [shadow including roots](https://dom.spec.whatwg.org/#concept-shadow-including-root) are [fully active](https://html.spec.whatwg.org/#fully-active); and
+- the [browsing contexts](https://html.spec.whatwg.org/#browsing-context) of their [shadow including roots](https://dom.spec.whatwg.org/#concept-shadow-including-root) either have a common [ancestor browsing context](https://html.spec.whatwg.org/#ancestor-browsing-context), or one is [ancestor browsing context](https://html.spec.whatwg.org/#ancestor-browsing-context) of the other, or are the same.
+
+A _set of browsing context related nodes_ is the set of all [nodes](https://dom.spec.whatwg.org/#concept-node) that are browsing context related with a [node](https://dom.spec.whatwg.org/#concept-node) whose [shadow including root]'s [browsing context](https://html.spec.whatwg.org/#browsing-context) is a [top-level browsing context](https://html.spec.whatwg.org/#top-level-browsing-context).
```

Comment:
I prefer 'a' since I see the definition as "a set of BCR nodes among all the nodes on the internet" in which case there is no unicity of top-level browsing context. If it is "a set of BCR nodes among all nodes in that browsing context tree" (where top-level browsing context is unique and we can use 'the'), then it's just the set of all nodes and does not really need to be that complex.

It is also more consistent with the use in the specs, _e.g._ the definition of [fully active](https://html.spec.whatwg.org/multipage/browsers.html#fully-active) uses "…its browsing context is a top-level browsing context" and https://html.spec.whatwg.org/multipage/browsers.html#security-nav is also using 'a' rather than 'the'.